lib/src/main/java/org/team100/lib/config/NotePoseDetector.java

```diff
@@ -34,8 +28,8 @@ public NoteDetector(
      * @return A robot relative translational x value of an object in a camera in
      *         meters
      */
-    public double getX() {
-        return m_cameraAngles.getX(m_notePosition24ArrayListener.getY().get());
+    public Double getX() {
```

Comment:
remove this.  callers can just get the Translation2d and pick out the x and y if they want to.